cmd/finch/main_native.go

```diff
@@ -79,6 +79,9 @@ var newApp = func(
 	// TODO: Decide when to forward --debug to the dependencies
 	// (e.g. nerdctl for container commands and limactl for VM commands).
 	rootCmd.PersistentFlags().Bool("debug", false, "running under debug mode")
+	rootCmd.PersistentFlags().String("namespace", "", `containerd namespace, such as "moby" for Docker, "k8s.io" for Kubernetes`)
+	rootCmd.PersistentFlags().String("cgroup-manager", "", `Cgroup manager to use ("cgroupfs"|"systemd")`)
```

Comment:
Do we actually support cgroupfs cgroup manager?

Reply:
> Do we actually support cgroupfs cgroup manager?

I was under the assumption that nerdctl handles this under the hood, I was just referencing [this](https://github.com/containerd/nerdctl/blob/cabd5bfef2f63ecf29d61a0fd50a740e5d359bc0/cmd/nerdctl/main.go#L120)

Apologies for the delayed response, I thought I replied here.